init-sensor.h

```diff
@@ -42,6 +42,9 @@ void initSensorType() {
     case SENSOR_DS18B20:
       initDallasSensor();
       break;
+    case SENSOR_ANALOG_TEST:
```

Comment:
You should init 0dB attenuation also for SENSOR_GUT800 .
Maybe we should set attenuation like configurable parameter for the projects which use ADC as well

Reply:
Do we need to configure attenuation only or rest ADC-related parameters too?

Reply:
No. Attenuation is the main parameter. But we can reserve code/memory for other possible usage of ADC related parameters: for.ex. ADC resolution

Reply:
DS also has resolution
It looks like we need to have sensor configurations and those configurations may be different for different sensors